spectator-api/src/test/java/com/netflix/spectator/api/CompositeRegistryTest.java

```diff
@@ -238,6 +238,18 @@ public void testAddAndRemove() {
   }
 
   @Test
+  public void testHasExpired() {
+    CompositeRegistry r = new CompositeRegistry(clock);
+
+    Counter c1 = r.counter("id1");
+    Assert.assertTrue(c1.hasExpired());
+
+    Registry r1 = new DefaultRegistry(clock);
+    r.add(r1);
+    Assert.assertTrue(c1.hasExpired());
```

Comment:
As long as you are here you might add one more assertion that c1.hasExpired returns false after you add a counter to r1.